deps/amqp10_client/test/system_SUITE.erl

```diff
@@ -324,61 +347,107 @@ roundtrip_large_messages(Config) ->
     DataMb = rand:bytes(1024 * 1024),
     Data8Mb = rand:bytes(8 * 1024 * 1024),
     Data64Mb = rand:bytes(64 * 1024 * 1024),
-    ok = roundtrip(OpenConf, DataKb),
-    ok = roundtrip(OpenConf, DataMb),
-    ok = roundtrip(OpenConf, Data8Mb),
-    ok = roundtrip(OpenConf, Data64Mb).
+    ok = roundtrip(OpenConf, [{body, DataKb}]),
+    ok = roundtrip(OpenConf, [{body, DataMb}]),
+    ok = roundtrip(OpenConf, [{body, Data8Mb}]),
+    ok = roundtrip(OpenConf, [{body, Data64Mb}]).
+
+basic_roundtrip_ibmmq(Config) ->
+    application:start(sasl),
```

Comment:
Why do we need to start the Erlang `sasl` application here?
Is https://www.erlang.org/doc/apps/sasl/error_logging.html being confused with https://datatracker.ietf.org/doc/html/rfc4422 ?

Reply:
I ran the test without this line, and the test succeeds.

Reply:
Indeed . In fact, it is not needed in the function `basic_roundtrip` either. I removed it from there and the tests succeeds.